labelme/app.py

```diff
@@ -1171,69 +1186,77 @@ def toggleDrawMode(self, edit=True, createMode="polygon"):
         self.canvas.setEditing(edit)
         self.canvas.createMode = createMode
         if edit:
-            if self._classType is None or 'segmentation' in self._classType:
+            if 'indoor' in self._classType:
                 self.actions.createMode.setEnabled(True)
-            else:
-                self.actions.createMode.setEnabled(False)
-            if self._classType is None or 'detection' in self._classType:
                 self.actions.createRectangleMode.setEnabled(True)
             else:
-                self.actions.createRectangleMode.setEnabled(False)
+                if self._classType is None or 'segmentation' in self._classType:
+                    self.actions.createMode.setEnabled(True)
+                else:
+                    self.actions.createMode.setEnabled(False)
+                if self._classType is None or 'detection' in self._classType:
+                    self.actions.createRectangleMode.setEnabled(True)
+                else:
+                    self.actions.createRectangleMode.setEnabled(False)
```

Comment:
```suggestion
                enable_drawing_polygon = self._classType is None or 'segmentation' in self._classType
                enable_drawing_rectangle = self._classType is None or 'detection' in self._classType
                self.actions.createRectangleMode.setEnabled(enable_drawing_rectangle)
                self.actions.createMode.setEnabled(enable_drawing_polygon)
```
위에서 구현하셨던것과 비슷하게 해도 되지 않을까요?